src/main/java/net/commments/sample/DifferentTraining.java

```diff
@@ -1,53 +1,53 @@
-package net.commments.sample;
-
-public class DifferentTraining {
-
-    public static void main(String[] args) {
-
-        String[] TeacherName = new String[2];
-        Teacher someTeacher_1 = new TeacherForStudent("Olga Petrova");
-        TeacherName[0] = someTeacher_1.name();
-        Teacher someTeacher_2 = new TeacherForStudent("Olga Ivanova");
-        TeacherName[1] = someTeacher_2.name();
-
-        String room[] = {"Room_1", "Room_2"};
-
-        System.out.println(TeacherName[0] + " teaches how to use git in " + room[0]);
-        System.out.println(TeacherName[1] + " demonstrates how to bake a cake in " + room[1]);
-
-
-        int[] places_1 = {0, 1, 2, 3, 4, 5, 6};
-
-        for (int i = 0; i < 1; i++) ;
-        {
-            System.out.println(room[0] + " contains unlimited palces" + " but " + room[1] + " has " + places_1[6] + " free places");
-        }
-        for (int i = 0; i < 2; i++) ;
-        {
-            System.out.println(room[0] + " contains unlimited palces" + " but " + room[1] + " has " + places_1[5] + " free places" + "(" + places_1[1] + " place is busy)");
-        }
-        for (int i = 0; i < 3; i++) ;
-        {
-            System.out.println(room[0] + " contains unlimited palces" + " but " + room[1] + " has " + places_1[4] + " free places" + "(" + places_1[2] + " places are busy)");
-        }
-        for (int i = 0; i < 4; i++) ;
-        {
-            System.out.println(room[0] + " contains unlimited palces" + " but " + room[1] + " has " + places_1[3] + " free places" + "(" + places_1[3] + " places are busy)");
-        }
-        for (int i = 0; i < 5; i++) ;
-        {
-            System.out.println(room[0] + " contains unlimited palces" + " but " + room[1] + " has " + places_1[2] + " free places" + "(" + places_1[4] + " places are busy)");
-        }
-        for (int i = 0; i < 6; i++) ;
-        {
-            System.out.println(room[0] + " contains unlimited palces" + " but " + room[1] + " has " + places_1[1] + " free place" + "(" + places_1[5] + " places are busy)");
-        }
-        for (int i = 0; i < 1; i++) ;
-        {
-            System.out.println(room[0] + " contains unlimited palces" + " We don’t learn a student if it is not in the " + room[1] + "(" + places_1[6] + " places are busy)");
-        }
-
-
-    }
-
-}
+//package net.commments.sample;
```

Comment:
@lvrub Please delete the file if you don't need this code, otherwise uncomment it.

Reply:
@extsoft  I've deleted commented code and pushed 
[Last commit](https://github.com/lvrub/Comments/commit/8f0f7120fb5e0c3d15030fa9acef2d040dc9498a)